modules/rules_abs/0.1.0/presubmit.yml

```diff
@@ -0,0 +1,12 @@
+bcr_test_module:
+  module_path: "."
+  matrix:
+    platform: ["debian10", "macos", "ubuntu2004"]
+    bazel: [7.x]
```

Comment:
can you test with 8.x as well?

Reply:
ah yes we actually verify this in CI. Added it here.